src/include/txn.h

```diff
@@ -60,10 +60,10 @@ struct __wt_txn_global {
 };
 
 typedef enum __wt_txn_isolation {
-	TXN_ISO_EVICTION,		/* Internal: eviction context */
-	TXN_ISO_READ_UNCOMMITTED,
-	TXN_ISO_READ_COMMITTED,
-	TXN_ISO_SNAPSHOT
+	WT_ISO_EVICTION,		/* Internal: eviction context */
+	WT_ISO_READ_UNCOMMITTED,
+	WT_ISO_READ_COMMITTED,
+	WT_ISO_SNAPSHOT
 } WT_TXN_ISOLATION;
```

Comment:
Note that we have public `WT_TXN_ISO_*` constants in the extension API as well as these private versions.  Ideally we would unify on the public names to avoid duplication, but I didn't try to make that change here.